hack/e2e-test.sh

```diff
@@ -169,7 +168,7 @@ go run "${SRC_ROOT}/contrib/cmd/hiveutil/main.go" create-cluster "${CLUSTER_NAME
 	--cloud="${CLOUD}" \
 	--creds-file="${CREDS_FILE}" \
 	--ssh-public-key-file="${SSH_PUBLIC_KEY_FILE}" \
-	--pull-secret-file="${PULL_SECRET_FILE}" \
+	--pull-secret-file="${PULL_SECRET_PATH}" \
```

Comment:
Although @staebler shouldn't this have worked because the env variable exists for CI?

Reply:
No. The env variable does not exist here.